daemon/response.go

```diff
@@ -223,6 +213,7 @@ func (ar assertResponse) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		}
 	}
+	w.WriteHeader(http.StatusOK)
```

Comment:
Calling `WriteHeader` after a `Write` as already happened (as with the `enc.Encode` call) will do nothing other than complain via a rather unfriendly log.
Given that as far as I can tell the code above can't return an error, why not call `WriteHeader` before the loop, instead of after?